python/cudf/cudf/tests/test_index.py

```diff
@@ -2565,3 +2565,84 @@ def test_rangeindex_max(rangeindex):
     expected = pidx.max()
 
     assert_eq(expected, actual)
+
+
+# RangeIndex tests that materializes the RangeIndex into int32 column under
+# user configuration.
```

Comment:
I appreciate the effort, but unfortunately this comment will be invalidated/become confusing as soon as someone adds another test below yours. Could you instead an explanatory docstring to each of the affected tests? They can basically be identically copy-pasted.